client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -245,6 +245,7 @@ def test_update_from_yaml_config(self, client, in_tempdir, created_endpoints, ex
         new_build_ids = get_build_ids(updated_status)
         assert len(new_build_ids) - len(new_build_ids.intersection(original_build_ids)) > 0
 
+    @pytest.mark.skip("bug in dev with env parameters")
```

Comment:
I believe Marc's fix to enable multiple environment variables has been merged. If it is fixed, could you verify, and un-skip this test? Thanks Evgenii.

Reply:
done